<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChipInfoWidget</class>
 <widget class="QWidget" name="ChipInfoWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLabel" name="label_chipName">
           <property name="text">
            <string>Chip Name:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_chipVersion">
           <property name="text">
            <string>Chip Version:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_extClock">
           <property name="text">
            <string>Ext Clock:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_ramType">
           <property name="text">
            <string>Extern RAM Type:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_ramSize">
           <property name="text">
            <string>Extern RAM Size:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_sramSize">
           <property name="text">
            <string>SRAM Size:</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLineEdit" name="lineEdit_chipName">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_chipVersion">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_extClock">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_ramType">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_ramSize">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_sramSize">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
